src/identity/reducers/index.ts

```diff
@@ -37,7 +39,11 @@ export const initialState: CurrentIdentity = {
     paygCreditStartDate: '',
     isUpgradeable: false,
   },
-  status: RemoteDataState.NotStarted,
+  loadingStatus: {
+    identityStatus: RemoteDataState.NotStarted,
+    billingStatus: RemoteDataState.NotStarted,
+    orgDetailsStatus: RemoteDataState.NotStarted,
+  },
```

Comment:
how do these properties map to the existing state? `identityStatus` seems to be related to `currentIdentity.user`, `orgDetailsStatus` seems to map to `currentIdentity.org`, but what is the relationship between `billingStatus` and `account`? Should these statuses should be part of the object they're meant to be a status for, rather than a separate object?

The thing I'm thinking about is that these statuses aren't actually part of the domain we're modelling. The loading status is a tool we use for mechanical reasons, not domain-modelling reasons; it's not related to the actual concept of an account or billing details.

Reply:
- `identityStatus` maps to the user's `currentIdentity`, retrieved from `/identity. 
- `billingStatus` maps to the `billingProvider` property, which is retrieved from /quartz/accounts/:accountId.  There's other stuff that comes back from this endpoint, but we don't use it at this point. So arguably this should be 'accountDetailsStatus' or something like that.
- `orgDetailsStatus` maps to the organization details retrieved from /quartz/accounts

I think it's right that this isn't related to an account or billing necessarily - the loading state imho is really a check for 'is it safe to load this page yet, or is there something else I need to do?' Maybe there's a better name here related to which pages we can load.

As far as keeping the status in the object - that's one possibility. However, I think it's safer to keep this in a separate object. Otherwise, changes to the loading status are going to trigger the selectors that rely on the retrieved data itself. 

I.e. - when you change the state to `LOADING` it'll trigger an unnecessary re-render of any page that's _using_ that data before the new data is retrieved.